README.md

```diff
@@ -95,7 +96,6 @@ In particular, applying gradients and background images to text is super underra
 - WTF Forms: Nicer Forms eg Radio Buttons with pure CSS https://github.com/mdo/wtf-forms
 - Output-inspired form: https://twitter.com/steveschoger/status/1171429842442522625
 
-
```

Comment:
This and `- spinner` below were auto-cleaned by built-in VS Code markdown formatter.

Let me know if you want to keep it as it is.